main.go

```diff
@@ -132,77 +133,78 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getMilestones(baseURL string, token string, projectID string) (map[string]string, error) {
-	milestones := []milestoneAPI{}
-
-	strURL := []string{baseURL, projectID, "/milestones?state=active&per_page=100"}
-	URL := strings.Join(strURL, "")
-
-	m, linkHeader, err := callServer(URL, token, milestones)
-	if err != nil {
-		return m, err
-	}
-
-	for _, milestone := range milestones {
-		m[milestone.Title] = milestone.DueDate
-	}
-
-	parsedHeader := link.Parse(linkHeader)
-	for _, elem := range parsedHeader {
-		if elem.Rel == "next" {
-			m, _, err := callServer(URL, token, milestones)
-			if err != nil {
-				return m, err
-			}
-			for _, milestone := range milestones {
-				m[milestone.Title] = milestone.DueDate
-			}
-		}
-	}
+func getActiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
+	state := "state=active"
```

Comment:
state should just be state := "active".
The state is not "state=active" that is actually mixing the state and the url structure.